src/i18n.ts

```diff
@@ -1,12 +1,19 @@
 import i18next, { type Resource } from 'i18next'
 import { prod } from './Config'
+import { DOMCacheGetOrSet } from './Cache/DOM';
 
-const supported = ['en', 'zh']
+// For 'flag': https://emojipedia.org/emoji-flag-sequence/
+// Searching "flag <country>" in their search bar will help verify the code.
+const supported: {[langID: string]: {name: string, flag: string}} = {
```

Comment:
```suggestion
const supported: Record<string, { name: string, flag: string }> = {
```
we want to eventually move to a stricter typescript lint

Reply:
Huh... how did I never run into the `Record` type until now?  It's been around long enough, according to the docs.